app/indexers/descriptive_metadata_datastream_indexer.rb

```diff
@@ -1,14 +1,18 @@
 # frozen_string_literal: true
 
 class DescriptiveMetadataDatastreamIndexer
-  attr_reader :resource
+  attr_reader :cocina
 
   def initialize(resource:, cocina:)
-    @resource = resource
+    @cocina = cocina.value!
```

Comment:
Why is a Success being passed instead of cocina?

Reply:
Success is a wrapper around the cocina, because not all objects have a cocina.  However after I rebase, this will be irrelevant.

Reply:
Irrelevant seems less confusing to me.